src/NzbDrone.Core/Notifications/Webhook/WebhookEpisodeFile.cs

```diff
@@ -20,6 +21,7 @@ public WebhookEpisodeFile(EpisodeFile episodeFile)
             SceneName = episodeFile.SceneName;
             Size = episodeFile.Size;
             DateAdded = episodeFile.DateAdded;
+            Languages = episodeFile.Languages.SelectList(language => language.Name).ConcatToString("/");
```

Comment:
CustomFormats, AudioLanguages, Subtitles, Images, etc are all lists so just a thought to make this a list to be consistent

Reply:
Agreed, we're not limited to simple strings like we are for scripts

Reply:
thanks for the feedback, swapped these over to Language lists

Reply:
Makes sense to also remove the ToString for Original language too.

I would also kinda agree that we could pass all these languages as list of language names (strings) instead of lists of language objects, like we do for custom formats for example.
Interested what the others think